src/Scope.spec.ts

```diff
@@ -791,7 +920,7 @@ describe('Scope', () => {
 
     describe('getDefinition', () => {
         it('returns empty list when there are no files', () => {
-            let file = program.addOrReplaceFile({ src: `${rootDir}/source/main.brs`, dest: 'source/main.brs' }, '');
+            let file = program.addOrReplaceFile({ src: `${rootDir} /source/main.brs`, dest: 'source/main.brs' }, '');
```

Comment:
This added a space in the path, which is completely bogus. Do you think you could go through and undo these additional unnecessary whitespace changes in this file?